fix: LEAP-580: Display non-string values in Text

robot-ci-heartex opened the pull request:
Hi @hlomzik!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7615343784) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1675

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7615343784

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7615364630

hlomzik committed:
Merge branch 'develop' into 'fb-leap-580/non-string-values'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7619698450

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7619698450

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7619716795

juliosgarbi reviewed (approved):
I saw that there are more files than it should be, but probably it's because someone merged the LSF PR but didn't merge the LSO PR.

hlomzik committed:
Merge branch 'develop' into 'fb-leap-580/non-string-values'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7625862773

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7625862773

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7625886659

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7628113831
